### ISSUE TITLE
✔️ 🐞 Bug Report: Extra SQL queries on every page. 

### ISSUE THREAD

**Issue by burhandodhy** [elementor/elementor#12162]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

Elementor run around 13 MySQL queries for the Lightbox settings on every page load and some other including `elementor_exclude_user_roles ` , `elementor_pro_facebook_app_id`, `medium_large_crop ` and the list go on...

As the rows are not available in options table WordPress cant cache the option values and eventually run the SQL queries. 

My suggestion is to save the default settings in the database so WordPress can cache them and this will eventually save the database calls.

![image](https://user-images.githubusercontent.com/7139602/90626376-4f031400-e234-11ea-870d-01b4b50b3ddb.png)

## Steps to reproduce

1) Install Elementor
2) Monitor MYSQL queries on every load. You will find queries likes this 
`SELECT option_value
FROM wp_options
WHERE option_name = 'elementor_lightbox_description_src'
LIMIT 1`

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by blueprintmrk** [comment]
This isn’t going to be fixed but it could become much faster. 

This plug-in requires js & css sadly you cannot ass something like this with out slowing your site speed

**Comment by Camilo517** [comment]
and slow queries 
http://prntscr.com/u8giam

**Comment by KingYes** [comment]
Hi @blueprintmrk,
The queries are by the WordPress Way. At this point in the process, Elementor cannot know whether the options are exits.

The idea of storing the cache of values I don't think it will be a good idea.

We are working to optimize the memory management and the queries, so maybe later, we can fix it in another way.

**Comment by KingYes** [comment]
@Camilo517,
Can you explain how to get this slow query in my environment?

**Comment by Camilo517** [comment]
@KingYes 
This plugin has many functionalities to detect errors
https://es.wordpress.org/plugins/query-monitor/
Unnecessary, duplicated, slow queries etc.

For example, I don't understand why these queries are made in a simple page with a form.

SELECT option_value
FROM cr6wPeGq_options
WHERE option_name = 'elementor_enable_inspector'
LIMIT 1

SELECT option_value
FROM cr6wPeGq_options
WHERE option_name = 'elementor_maintenance_mode_mode'
LIMIT 1

SELECT option_value
FROM cr6wPeGq_options
WHERE option_name = 'elementor_exclude_user_roles'
LIMIT 1

SELECT option_value
FROM cr6wPeGq_options
WHERE option_name = 'elementor_icon_manager_needs_update'
LIMIT 1

**Comment by nicholaszein** [comment]
Hello everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor **v3.9.0**! 🥳 

Thank you for your feedback! 🙏

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
